lib/SignIn.dart

```diff
@@ -0,0 +1,266 @@
+                                    right:
+                                        MediaQuery.of(context).size.width * .1),
+                                child: const Text(
+                                  "Forgot Password",
+                                  style: TextStyle(
+                                      fontWeight: FontWeight.bold,
+                                      fontSize: 15,
+                                      color: Colors.white),
+                                ),
+                              ),
+                            ),
+
+                            InkWell(
+                                onTap: () {
+                                  Navigator.push(
+                                    context,
+                                    MaterialPageRoute(
+                                        builder: (context) =>
+                                            const MyHomePage()),
+                                  );
+                                },
+                                child: const Text(
+                                  "Create a New Account",
+                                  style: TextStyle(
+                                      fontWeight: FontWeight.bold,
+                                      fontSize: 15,
+                                      color: Colors.white),
+                                )),
+                          ],
+                        ),
+                        SizedBox(
+                          height: 20,
+                        ),
+                        MaterialButton(
+                          color: Color.fromRGBO(0, 0, 0, 0.3),
+                          minWidth: MediaQuery.of(context).size.width * .8,
+                          height: 55,
+                          shape: RoundedRectangleBorder(
+                              borderRadius:
+                                  BorderRadius.all(Radius.circular(20.0))),
+                          onPressed: () {
+                            if (_formKey.currentState!.validate()) {
+                              // Process data.
+                            }
+                          },
+                          child: Text(
+                            "Sign In",
+                            style: TextStyle(color: Colors.white),
+                          ),
+                        ),
+                        SizedBox(
+                          height: 70,
+                        ),
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          crossAxisAlignment: CrossAxisAlignment.end,
+                          children: [
+                            FloatingActionButton(
+                              heroTag: '1',
+                              onPressed: () {},
+                              child: Image.asset(
+                                  "asset/image/Google_Plus_icon_(2015-2019).svg.png"),
```

Comment:
Please do not use an image path like this. Instead, you can create a separate class and mention the path of the images there and use it here.